pkg/inject/offset_results.json

```diff
@@ -5,2556 +5,2772 @@
       "data_members": [
         {
           "struct": "net/http.Request",
-          "field_name": "RemoteAddr",
+          "field_name": "Method",
           "offsets": [
             {
-              "offset": 176,
+              "offset": 0,
```

Comment:
Maybe I'm missing something, but why have these offsets changed for existing versions? (the same for `field_name` above)?

Reply:
This can change based on the Go version or the field is restructured.